README.md

```diff
@@ -1,2 +1,2 @@
 # py-commons
-Common Python Library 
+This common python library is dedicated to metadata matching and finding regressions.
```

Comment:
This line might be more appropriate in the fmatch directory.

Reply:
Thank you for the suggestion Joe. I'll add the readme to fmatch.